cluster/gce/util.sh

```diff
@@ -899,6 +899,7 @@ ENABLE_NODE_JOURNAL: $(yaml-quote ${ENABLE_NODE_JOURNAL:-false})
 PROMETHEUS_TO_SD_ENDPOINT: $(yaml-quote ${PROMETHEUS_TO_SD_ENDPOINT:-})
 PROMETHEUS_TO_SD_PREFIX: $(yaml-quote ${PROMETHEUS_TO_SD_PREFIX:-})
 ENABLE_PROMETHEUS_TO_SD: $(yaml-quote ${ENABLE_PROMETHEUS_TO_SD:-false})
+DISABLE_PROMETHEUS_TO_SD_IN_DS: $(yaml-quote ${DISABLE_PROMETHEUS_TO_SD_IN_DS:-false})
```

Comment:
Generally these parameters are also exposed in config-default.sh and config-test.sh so that people know that they can be toggled.

Reply:
We do not expect people to disable prometheus-to-sd for DaemonSets only. For GKE specifically our team wants to monitor DaemonSets with just one prometheus-to-sd deployed as DaemonSet, thus this is why we are adding this flag.